src/documentation/SidebarContent/EmptyContent.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function EmptyContent() {
+    return (
+        <div style={{display:'flex',flexDirection:'column', justifyContent:'center', alignItems:'center', backgroundColor:'#edf1f3'}}>
```

Comment:
https://uikit.wfp.org/next-docs/index.html?path=/docs/components-ui-elements-empty--regular

Reply:
Yes, I will replace it with the empty one you created